src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -859,7 +876,8 @@
       <sch:let name="languageTranslated_present" value="geonet:values-in($languageTranslated,
               ('eng', 'fra', 'spa', 'zxx'))"/>
 
-      <sch:let name="locMsgCt" value="geonet:prependLocaleMessage($loc/strings/ResourceDescriptionContentType, concat(gmd:CI_OnlineResource/gmd:linkage/gmd:URL, ' : '))" />
+      <sch:let name="resourceContentTypesList" value="geonet:resourceContentTypesList($thesaurusDir,$altLanguage2char)"/>
```

Comment:
For a metadata with main language English, the values displayed are always in French, doesn't matter if the content type invalid value is in English or French:

> Online resource description content type is not valid. Valid values are: API, Application, Données, Document de soutien, Service Web

I think the correct way is to retrieve the message for English and French and depending which value fails, use one or another.

Reply:
Updated based on suggestion
Added message for both English and French Validation
Also updated similar logic in non-multilingual.sch to use thesaurus values in the assertion